.gitignore

```diff
@@ -2,3 +2,6 @@ db.sqlite3
 __pycache__/
 !media/temp/
 media/
+caropa_app/migrations/__pycache__/*
+caropa_app/migrations/*
+!caropa_app/migrations/__init__.py
```

Comment:
Done!

Reply:
Remove: `caropa_app/migrations/__pycache__/*`
Change: `caropa_app/migrations/*` to `caropa_app/migrations/`


To push the file: `caropa_app/migrations/__init__.py`, run `git add caropa_app/migrations/__init__.py -f`
The reason it's because by default gitignore is gonna ignore `caropa_app/migrations/__init__.py`

Reply:
Apologise, In the last commit I haven't push the changes on gitignore